task-list/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="page_element">
+<h1>Task List</h1>
+
+  <table>
+    <thead>
+      <tr>
+        <th colspan="3">
+          <h2>Your Complete Task List</h2>
+        </th>
+      </tr>
+      <tr>
+        <th>
+
+        </th>
+        <th>
+          Task
+        </th>
+        <th>
+          Completed Date
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tasks.each do |task| %>
+      <tr>
+        <td class="bigger_checkbox">
+          <% if task.completed_date.nil? %>
+            ☐
+            <% @class = "incomplete" %>
+          <% else %>
+            ☑
+            <% @class = "complete" %>
```

Comment:
I think that `@class` should be a local variable not an instance variable because it doesn't need to be accessed outside of view